data/categories.txt

```diff
@@ -6801,6 +6950,117 @@ vi:Quần vợt
 zh-Hans:网球
 zh-Hant:網球
 
+sport-volleyball
+en:Volleyball
+ar:كرة الطائرة
+be:Валейбол
+bg:Волейбол
+cs:Volejbal
+da:Volleyball
+de:Volleyball
+el:Πετοσφαίριση
+es:Voleibol
+eu:Boleibol
+fa:والیبال
+fi:Lentopallo
+fr:Volley-ball
+he:כדורעף
+hu:Röplabda
+id:Bola voli
+it:Pallavolo
+ja:バレーボール
+ko:배구
+mr:व्हॉलीबॉल
+nb:Volleyball
+nl:Volleyball
+pl:Piłka siatkowa
+pt:Voleibol
+ro:Volei
+ru:Волейбол
+sk:Volejbal
+sv:Volleyboll
+sw:Voliboli
+th:วอลเลย์บอล
+tr:Voleybol
+uk:Волейбол
+vi:Bóng chuyền
+zh-Hans:排球
+zh-Hant:排球
+
+sport=9pin
+en:Nine-pin Bowling
```

Comment:
Does it make sense to differentiate these two types? Let's merge them into one to avoid copy-paste. CC @vng

Reply:
Well, yes, I'd merge them in categories like:
sport-9pin|sport-10pin


And in types_strings like:
```
[type.sport.9pin]
  ref = type.sport.10pin
```